dashboard/src/components/AppView/AppControls/StatusAwareButton.tsx

```diff
@@ -0,0 +1,32 @@
+import { CdsButton } from "@cds/react/button";
+import ReactTooltip from "react-tooltip";
+import { hapi } from "../../../shared/hapi/release";
+
+export interface IStatusAwareButtonProps {
+  id: string;
+  releaseStatus: hapi.release.IStatus | undefined | null;
```

Comment:
In other places, you are using `status`; I would suggest renaming `status` to `releaseStatus` instead. IMO, just using `status` reminds me of a true/false value instead of a `hapi.release.IStatus` object.